crowbar_framework/app/models/trove_service.rb

```diff
@@ -25,7 +25,10 @@ def role_constraints
       {
         "trove" => {
           "unique" => false,
-          "count" => -1
+          "count" => -1,
```

Comment:
Not really related to this PR, but I'm unsure that the count constraint is correct :/